src/www/interfaces_vlan_edit.php

```diff
@@ -200,6 +206,31 @@
                       </div>
                     </td>
                   </tr>
+<?
+$priorities = array(
+    1   => gettext('1 - Background'),
+    0   => gettext('0 - Best Effort (default)'),
```

Comment:
Except that is not the case, 0 is the default, it will always be selected when no existing value is set (NULL == 0). It's already 0 in ifconfig as well.